kinto_http/exceptions.py

```diff
@@ -15,3 +15,10 @@ def __init__(self, message=None, exception=None):
 
 class BucketNotFound(KintoException):
     pass
+
+
+class BackoffException(KintoException):
+
+    def __init__(message, retry_after):
```

Comment:
Since you inherit from `KintoException`, you might want to keep the `exception=None` kwarg